### ISSUE TITLE
Can wasmedge now support model training?

### ISSUE THREAD

**Issue by 17862687921** [WasmEdge/WasmEdge#2818]
Can wasmedge now support model training?

**Comment by hydai** [comment]
Are you talking about AI model training? If so, do you have any use cases?

**Comment by 17862687921** [comment]
Yes. You can take Cifar-10 model training as an example, use the AlexNet model as a pre-training model, and train the model through the Cifar-10 data set.

**Comment by hydai** [comment]
Hi @17862687921 
I am afraid that we don't have such a plan for supporting model training now. The reason is that WASI-NN doesn't define and is not designed for model training. Instead of using WASI-NN, we have to devise a solution for the model training. However, there is a big issue: what kinds of APIs do we need? We need a clear answer to this. So, we are open to talk and welcome any contribution to this feature.

**Comment by 17862687921** [comment]
@hydai Hello, I want to use pytorch or tensorflow to implement model training. Similar to python, model training can be completed by directly giving the data set, hyperparameters and training code (including model architecture).
